app/interactors/create_user.rb

```diff
@@ -1,9 +1,9 @@
 class CreateUser
-  include Interactor
+  include Interactor::Organizer
 
-  def call
-    context.user = User.new(context.user_params)
-    context.fail!(errors: context.user.errors) unless context.user.save
-    UserMailer.registration_confirmation(context.user).deliver_now
-  end
+  organize MakeNewUser,
```

Comment:
I think we might want to organize this a bit differently. Right now, `EncryptPassword`, `SaveUser`, and `CreateConfirmationToken` all make different request to the database. It might not be a big deal right now but for an app with a lot of traffic it can add up fairly quickly :smiley:. 

We can talk a bit more about how to set it up. I would have split this into making the new user and sending the confirmation token but I also like your granular approach. I think the way in which we can split up interactors is mostly arbitrary and up to personal choice. We just have to make sure there is no performance impact. :+1: :smile: